.drone.yml

```diff
@@ -3119,6 +3075,68 @@ steps:
         make change-amis-to-public-ent
         make change-amis-to-public-ent-fips
 
+  # all mandatory steps for a release promotion need to go BEFORE this step,
+  # as there is a chance that everything afterwards will be skipped
+  - name: Skip RPM/DEB publishing for alpha/beta/dev/rc builds
+    image: docker
+    commands:
+      - |
+        # length will be 0 after filtering if this is a 'dev' release, >0 otherwise
+        FILTERED_TAG_LENGTH=$(echo ${DRONE_TAG} | egrep -v '(alpha|beta|dev|rc)' | wc -c)
+        if [ $$FILTERED_TAG_LENGTH -eq 0 ]; then
+          echo "---> ${DRONE_TAG} looks like a dev release, not publishing RPMs"
+          # exit pipeline early with success status
```

Comment:
Add a link to Drone docs explaining this magic exit status like you did in the PR description?

Reply:
https://gravitational.slab.com/posts/drone-ci-bdbtz26a#what-does-exit-78-do